src/OutputBox.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Button, TextField } from '@mui/material';
+
+export default () => {
+  return (
+    <>
+      <div id='Output'>
+        <TextField
+          multiline
+          rows={20}
+          placeHolder='Plain English'
```

Comment:
I believe this should be `placeholder` instead of `placeHolder` (note caps of letter h).